.dockerignore

```diff
@@ -0,0 +1,15 @@
+.*cache
+.DS_Store*
+*.egg*
+.idea
+*.lock
```

Comment:
This one will bite us when we introduce poetry.lock files. Maybe remove the *.lock from ignore?